kivy/logger.py

```diff
@@ -17,17 +25,40 @@
     except Exception:
         Logger.exception('Something happened!')
 
-The message passed to the logger is split into two parts, separated by a colon
-(:). The first part is used as a title, and the second part is used as the
-message. This way, you can "categorize" your message easily. ::
+The message passed to the logger is split into two parts separated by a colon
+(:). The first part is used as a title and the second part is used as the
+message. This way, you can "categorize" your messages easily. ::
 
     Logger.info('Application: This is a test')
 
     # will appear as
 
     [INFO   ] [Application ] This is a test
 
-Logger configuration
+You can change the logging level at any time using the `setLevel` method. ::
+
+    from kivy.logger import Logger, LOG_LEVELS
+
+    Logger.setLevel(LOG_LEVELS["debug"])
+
+
+Features
+--------
+
+Although you are free to use standard python loggers, the Kivy `Logger` offers
+some solid benefits and useful features. These include:
+
+* simplied usage (single instance, simple configuration, works by default)
+* color-coded output
+* output to `stdout` by default
+* message categorization via colon separation
+* access to log history even if logging is disabled
+* built-in handling of various cross-platform considerations
+
+For these reasons, we advise that you use the Kivy Logger unless you have good
```

Comment:
I'm hesitant to push for it that strongly. I would rather state it indirectly - "that kivy's logger was designed to be used with kivy apps and it makes logging from such app more convenient", and leave it at that.

Reply:
Revised accordingly. Thanks

Reply:
@matham Bump :-)